react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -149,6 +149,10 @@ public boolean isContentChanged() {
         void onEditorAutosave();
     }
 
+    public interface OnPreferredColorSchemeListener {
+        String getPreferredColorScheme();
+    }
```

Comment:
Not a big deal, but "Listener" doesn't quite seem right here to me. I think of a "Listener" object as something that has events/updates pushed to it and then responds to those events, not something that provides those updates to others on demand like this interface does. 

To put it another way, I would expect a `PreferredColorSchemeListener` interface would be something like this:

```
   public interface PreferredColorSchemeListener {
        void onColorSchemeChanged(String colorScheme);
    }
``` 

Renaming this interface from `OnPreferredColorSchemeListener` to something like `PreferredColorSchemeSource` or `PreferredColorSchemeProvider` would seems a bit more accurate to me (I'm not remotely invested in either of those suggestions though).

Reply:
Same here. As I have changed a lot of code we would need another iteration of code review.